### PR TITLE
Add connector bulk import feature

### PR DESCRIPTION
**Pull request by shyamir**

### CODE REVIEW

File: plugins/org.wso2.developerstudio.eclipse.esb.project/src/org/wso2/developerstudio/eclipse/esb/project/ui/wizard/ConnectorFromFileSystemDialog.java [hunk 1]
```diff
@@ -107,10 +112,22 @@ public void modifyText(ModifyEvent evt) {
...
                 if (fileName != null) {
-                    txtCloudConnectorPath.setText(fileName);
+                    // Store the file names in an array
+                    String[] connectorPaths = fileDlg.getFileNames();
+                    String displayText = "";
+
+                    for (int i=0; i<connectorPaths.length;i++) {
+                        if (i!=0) {
+                            displayText += ", " + fileDlg.getFilterPath() + File.separator + connectorPaths[i];
+                        }else {
```
Comment (sameeragunarathne) [inline plugins/org.wso2.developerstudio.eclipse.esb.project/src/org/wso2/developerstudio/eclipse/esb/project/ui/wizard/ConnectorFromFileSystemDialog.java:126]: formatting issue

Comment (sameeragunarathne) [inline plugins/org.wso2.developerstudio.eclipse.esb.project/src/org/wso2/developerstudio/eclipse/esb/project/ui/wizard/ConnectorFromFileSystemDialog.java:126]:
```suggestion
                        } else {
```

File: plugins/org.wso2.developerstudio.eclipse.esb.project/src/org/wso2/developerstudio/eclipse/esb/project/ui/wizard/ConnectorFromFileSystemDialog.java [hunk 2]
```diff
@@ -107,10 +112,22 @@ public void modifyText(ModifyEvent evt) {
...
+
                 if (fileName != null) {
-                    txtCloudConnectorPath.setText(fileName);
+                    // Store the file names in an array
+                    String[] connectorPaths = fileDlg.getFileNames();
+                    String displayText = "";
+
+                    for (int i=0; i<connectorPaths.length;i++) {
+                        if (i!=0) {
+                            displayText += ", " + fileDlg.getFilterPath() + File.separator + connectorPaths[i];
```
Comment (prabushi) [inline plugins/org.wso2.developerstudio.eclipse.esb.project/src/org/wso2/developerstudio/eclipse/esb/project/ui/wizard/ConnectorFromFileSystemDialog.java:125]: Shall we move fileDlg.getFilterPath() + File.separator to a variable before the for loop?

File: plugins/org.wso2.developerstudio.eclipse.esb.project/src/org/wso2/developerstudio/eclipse/esb/project/ui/wizard/ConnectorFromFileSystemDialog.java [hunk 3]
```diff
@@ -1,20 +1,20 @@
 /*
-*  Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (sameeragunarathne) [inline plugins/org.wso2.developerstudio.eclipse.esb.project/src/org/wso2/developerstudio/eclipse/esb/project/ui/wizard/ConnectorFromFileSystemDialog.java:2]: Licence header has been formatted. let's keep it as it is

File: plugins/org.wso2.developerstudio.eclipse.esb.project/src/org/wso2/developerstudio/eclipse/esb/project/ui/wizard/CloudConnectorImportWizard.java [hunk 4]
```diff
@@ -86,17 +86,19 @@ public boolean performFinish() {
...
             if (!parentDirectory.exists()) {
                 parentDirectory.mkdir();
             }
-            File file = new File(source);
-            FileUtils.copyFileToDirectory(file, parentDirectory);
-
-            updateProjects(source);
+            for (String value : values) {
+                File file = new File(value);
+                FileUtils.copyFileToDirectory(file, parentDirectory);
```
Comment (prabushi) [inline plugins/org.wso2.developerstudio.eclipse.esb.project/src/org/wso2/developerstudio/eclipse/esb/project/ui/wizard/CloudConnectorImportWizard.java:99]: Shall we use Files.copy without creating new File objects?

**Review by sameeragunarathne — APPROVED** [review]

**Review by prabushi — APPROVED** [review]

**Pull request merged**